apps/tangle-dapp/components/charts/types.ts

```diff
@@ -17,12 +15,10 @@ export interface PieChartProps {
   title?: string;
 }
 
-type EarningsByServiceType = Partial<Record<RestakingService, number>>;
-
 export type RoleEarningsChartItem = {
-  month: string;
-  year: number;
-} & EarningsByServiceType;
+  era: number;
```

Comment:
The data for the Roles Earnings Chart should be shown by time (month), not era, right? I think the app will be more user-friendly if we show earnings by month (or day) instead of by era. 

However, this cannot be done at the moment because the Graphql for Tangle is not deployed yet

Reply:
Yes, currently I haven't found any way to convert the era into a timestamp, so I'm using the era number instead.